apps/common/lib/config/config.ts

```diff
@@ -1,7 +1,7 @@
 const config = {
   appEnv: process.env.APP_ENV || "test",
   authAllowedGroups: process.env.AUTH_ALLOWED_GROUPS?.split(",") || ["TEST_GROUP"],
-  authDomain: process.env.AUTH_DOMAIN || "//auth.hackney.gov.uk/auth",
+  authDomain: process.env.AUTH_DOMAIN || "//auth-test.hackney.gov.uk/auth",
```

Comment:
Should we just pass the `AUTH_DOMAIN` environment variable in our development environment? Do we need to update the default instead?

Reply:
I was going to update the AUTH_DOMAIN environment variable but the parameter store entry isn't there.  Since we are just doing a test and revert on this occasion (as with MAA) I felt the quickest step would be to update the default then roll back once the tests were successful.

Reply:
Upon further investigation, It appears that the auth url is not take nfrom the environment varialbe but is hard coded in a local.ts file.  This has now been updated.